superset-frontend/src/components/ExpandableList.tsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { ReactNode, useState } from 'react';
+// @ts-ignore
+import { Button } from 'react-bootstrap';
+import { css } from '@emotion/core';
+
+interface Props {
+  items: string[] | ReactNode[];
+  display?: number;
+}
+
+function intersperse(arr: any[], sep: string | ReactNode) {
+  if (arr.length === 0) {
+    return [];
+  }
+
+  return arr.slice(1).reduce((xs, x) => xs.concat([sep, x]), [arr[0]]);
+}
+
+const ActionButton: React.FunctionComponent<{ onClick: () => void }> = ({
+  onClick,
+  children,
+}) => (
+  <Button
+    bsStyle="link"
+    css={css`
```

Comment:
These little ad hoc css tweaks, in the long run, will make me nervous. At some point, we'll need to make the super-awesome `SupersetButton` that does everything we need it to with props, so we don't need to pepper overrides around the codebase. Not a big concern for this PR, but pointing out some hopes and dreams :)

Reply:
yea we need a `<Button as="Link"/>` or something similar. I'll move this to `src/components`